### PR TITLE
make errors

### PR DESCRIPTION
**Pull request by Fallatori**
I made error for testing

### CODE REVIEW

File: src/SideFlare.tsx [hunk 1]
```diff
@@ -12,12 +12,10 @@ const Diagram = styled.div``
 
 const Decoding = styled.div``
 
-const HardDrive = styled.div``
```
Comment (Materie) [inline src/SideFlare.tsx]: This is in use, why is it removed?

File: src/SideFlare.tsx [hunk 2]
```diff
@@ -12,12 +12,10 @@ const Diagram = styled.div``
...
-
 export const SideFlare = () => {
 	return (
 		<Wrapper>
-			<Diagram>Diagram</Diagram>
+			<Diagram>Decoding!!!!</Diagram>
```
Comment (Materie) [inline src/SideFlare.tsx]: Is this correct? The text seems to be copied from the below element.

**Review by Materie — CHANGES_REQUESTED** [review]
A few problems here.

File: src/SideFlare.tsx (hunk 1 above)
Comment (Fallatori) [reply src/SideFlare.tsx]: I have fixed it

**Review by Materie — APPROVED** [review]
Looks good ;)

**Pull request merged**